packages/core/docs/performance/good-practices.md

```diff
@@ -0,0 +1,338 @@
+
+### Reduce third-party usage ⚫⚫⚫
+
+Think twice you really need it, if yes don't load it on pages where they will not be used (eg. reCAPTCHA  on the home page if there is no form).
+Third-party code can significantly impact load performance. You can however modify the way you are using this third party library.
+
+::: tip HINTS
+- Load scripts using the async or defer attribute to avoid blocking document parsing.
+- Self-host the script if the third-party server is slow.
+- Remove the script if it doesn't add clear value to your site.
+- Use link rel=preconnect or link rel=dns-prefetch to perform a DNS lookup for domains hosting third-party scripts.
+- You can lazy load third-party resources [with facades](https://web.dev/third-party-facades/?utm_source=lighthouse&utm_medium=devtools)
+- Move third parties to web worker using eg Partytown, you can do it easily with VSF using nuxt [Partytown module](https://github.com/nuxt-community/partytown-module)
+:::
+
+## Optimizing images
+
+### Keep image size small and serve them in next gen formats  ⚫⚫⚫
+
+You should always compress images, there is lot of lossless compression file types supported with all modern browsers (eg. webp). This is the most common performance bottleneck, where images are not compressed and weigh multiple times more than they could if compressed.
+
+::: tip HINTS
+- You can use nuxt-image library to automatically transform images on the fly to next gen formats and compress them using one of many available providers (eg. Cloudinary or local instance of ipx)
+:::
+
+### Avoid loading images as a background images above the fold  ⚫⚫⚫
+
+Images declared in the CSS files are often downloaded later because first, proper css needs to be downloaded and parsed.
+
+::: tip HINTS
+- If possible, declare the biggest images visible on the page in an HTML document without lazy loading instead of in CSS files.
+:::
+
+### Lazy load offscreen images ⚫⚫⚫
+
+Lazy loading is a technique used to prevent unnecessary resources to be downloaded on the page render. Users get only images (or other types of resources, as videos or even whole components) that are shown in the current viewport. Rest is downloaded when needed, eg. if we have an image in the footer it will be downloaded only when User scrolls near it. 
+
+Only images above the fold should be loaded on page render, rest should be lazy loaded.
+Remember don't lazy load image that is potential candidate for Largest Contentful Paint.
+
+::: tip HINTS
+- If you are using SFUI components eg. SfImage, all images are lazy loaded automatically. But if you are using your own implementation remember to use 
+```html
+<img loading=lazy>
+```
+ or eg. nuxt-lazy-load library (depending on your needs and browser support).
+- If you want to use lazy loading in Nuxt 3 go with this [lazy-load module](https://github.com/nuxt-modules/lazy-load).
```

Comment:
Here as well, VSF is Nuxt 2 right now and will not probably be migrated to Nuxt 3 in 6 months at least. Maybe choose the current vue-lazyload module? It will work better for VSF

Reply:
Fixed